spec/helpers/errors_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the ErrorsHelper. For example:
+#
+# describe ErrorsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe ErrorsHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Tak w sumie to musimy zastanowić się czy ten plik jest w ogóle potrzebny, bo nie nic z niego nie jest wykorzystywane, to tylko przykłady